### PR TITLE
Fix block types select width and overflowing form width in Firefox

### PR DESCRIPTION
**Pull request by tniebergall**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Set the width of the types select field to 150px to also show longer block names

Bildschirmfoto 2020-07-11 um 23 17 06

Bildschirmfoto 2020-07-11 um 23 16 49

#### Why?

To show longer block names in full length

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/block.scss [hunk 1]
```diff
@@ -72,7 +72,7 @@ $blockIconsColor: $gray;
 }
 
 .types {
-    width: 120px;
+    width: 150px;
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/block.scss:75]: i think we can just use `min-width: 120px;` here. that would fix the problem for even longer type names

**Comment by nnatter** [comment]
Hey,
I rebased this PR and implemented the suggestions. Hope that is okay for you.
Thanks again for your effort!

**Comment by nnatter** [comment]
This can be tested with the following block property:
```xml
<block name="blocks" default-type="editor" minOccurs="0">
    <meta>
        <title lang="en">Content</title>
        <title lang="de">Inhalte</title>
    </meta>
    <types>
        <type name="editor">
            <meta>
                <title lang="en">Editor with long name</title>
                <title lang="de">Editor mit langem Namen</title>
            </meta>
            <properties>
                <property name="text_editor" type="text_editor">
                    <meta>
                        <title lang="en">Text Editor</title>
                        <title lang="de">Texteditor</title>
                    </meta>
                </property>
            </properties>
        </type>
        <type name="media">
            <meta>
                <title lang="de">Media Selection with even longer name than editor, probably so long that it does not fit on the screen anymore without cropping it</title>
                <title lang="en">Medienauswahl mit noch längerem Namen wie Editor, vermutlich so lang, dass er nicht mehr auf dem Display dargestellt werden kann ohne gecroppt zu werden</title>
            </meta>
            <properties>
                <property name="media_selection" type="media_selection">
                    <meta>
                        <title lang="en">Media Selection</title>
                        <title lang="de">Medienauswahl</title>
                    </meta>
                </property>
            </properties>
        </type>
    </types>
</block>
```

**Comment by alexander-schranz** [comment]
In Firefox the current solution didn't work I needed to adjust some more css that the Rendered Form is not going outside the container and the application view works correctly there.

**Pull request merged**